### ISSUE TITLE
Prefix user-facing interfaces with "I" to match .NET Convention

### ISSUE THREAD

**Issue by SeanKilleen** [akkadotnet/akka.net#652]
Per #633, this should likely be undertaken after #651.

**Comment by HCanber** [comment]
If this is to be done, it should be done before v1? Right?

**Comment by Aaronontheweb** [comment]
@HCanber yeah, we should just bit the bullet and do this now. 

Does anyone want this issue assigned to them? Should be straightforward.

**Comment by SeanKilleen** [comment]
I'll try to take this one on. I haven't gotten to really contribute yet but with that in mind I'm happy to give it a shot. :)

**Comment by HCanber** [comment]
@Aaronontheweb  Was this closed intentionally, or automatically by merging my PR? Because it isn't fixed.

**Comment by Aaronontheweb** [comment]
@HCanber whoops, I thought you said in your PR that this was finished. Looks like I misread it. My bad!

**Comment by HCanber** [comment]
These are the ones @nvivo listed in #633, and which have been fixed. Once this list is complete, we need to recheck and make sure there aren't any more.

- [X] `Akka.Actor.ActorRefProvider, Akka` (Public)
- [X] `Akka.Actor.ActorRefScope, Akka` (Public)
- [X] `Akka.Actor.AutoReceivedMessage, Akka` (Public)
- [X] `Akka.Actor.Cell, Akka` (Public)
- [X] `Akka.Actor.Inboxable, Akka` (Public)
- [X] `Akka.Actor.IndirectActorProducer, Akka` (Public)
- [X] `Akka.Actor.Internal.ChildrenContainer, Akka` (Public)
- [X] `Akka.Actor.Internal.ChildStats, Akka` (Public)
- [X] `Akka.Actor.Internal.InternalSupportsTestFSMRef`2, Akka` (Public)
- [X] `Akka.Actor.Internal.SuspendReason+WaitingForChildren, Akka`
- [X] `Akka.Actor.Internals.InitializableActor, Akka` (Public)
- [X] `Akka.Actor.LocalRef, Akka`
- [ ] `Akka.Actor.LoggingFSM, Akka` (Public)
- [ ] `Akka.Actor.NoSerializationVerificationNeeded, Akka` (Public)
- [ ] `Akka.Actor.PossiblyHarmful, Akka` (Public)
- [X] `Akka.Actor.RepointableRef, Akka` (Public)
- [ ] `Akka.Actor.WithBoundedStash, Akka` (Public)
- [ ] `Akka.Actor.WithUnboundedStash, Akka` (Public)
- [ ] `Akka.Dispatch.BlockingMessageQueueSemantics, Akka` (Public)
- [ ] `Akka.Dispatch.BoundedDequeBasedMessageQueueSemantics, Akka` (Public)
- [ ] `Akka.Dispatch.BoundedMessageQueueSemantics, Akka` (Public)
- [ ] `Akka.Dispatch.DequeBasedMailbox, Akka` (Public)
- [ ] `Akka.Dispatch.DequeBasedMessageQueueSemantics, Akka` (Public)
- [ ] `Akka.Dispatch.MessageQueues.MessageQueue, Akka` (Public)
- [ ] `Akka.Dispatch.MultipleConsumerSemantics, Akka` (Public)
- [ ] `Akka.Dispatch.RequiresMessageQueue`1, Akka` (Public)
- [ ] `Akka.Dispatch.Semantics, Akka` (Public)
- [ ] `Akka.Dispatch.SysMsg.SystemMessage, Akka` (Public)
- [ ] `Akka.Dispatch.UnboundedDequeBasedMessageQueueSemantics, Akka` (Public)
- [ ] `Akka.Dispatch.UnboundedMessageQueueSemantics, Akka` (Public)
- [ ] `Akka.Event.LoggingAdapter, Akka` (Public)
- [ ] `Akka.FluentConfigInternals, Akka` (Public)
- [ ] `Akka.Remote.InboundMessageDispatcher, Akka.Remote`
- [ ] `Akka.Remote.RemoteRef, Akka.Remote`
- [ ] `Akka.Routing.ConsistentHashable, Akka` (Public)

**Comment by HCanber** [comment]
I've done a few. Up for grabs for someone else. :)

**Comment by SeanKilleen** [comment]
Just noting: once #801 is merged in, this list will be complete I think and can be closed.

**Comment by SeanKilleen** [comment]
Closing; all interfaces listed in this issue have been accounted for and a new issue is opened for the additional items.
